### PR TITLE
Single video page

### PR DESCRIPTION
**Pull request by omkarpatke**
# Added Single Video Page

**ScreenShot : **
Screenshot 2022-05-05 155900

**Files To Check:**
- HomePage.js
- SingleVideo.js
- SingleVideo.css

### CODE REVIEW

File: src/Pages/HomePage/HomePage.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    flex-wrap: wrap;
+    flex-direction: row;
+    gap: 1rem 1rem;
+    background-color: var(--bg-color);
+    padding: 1rem;
+}
+
+.video{
+   background-color: var(--white-color);
+   max-width: 300px;
```
Comment (vaishnavipaliya11) [inline src/Pages/HomePage/HomePage.css]: can use rem.

File: src/Pages/HomePage/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { SideBar, CategoryBar } from '../../components/index';
+import { useVideos } from '../../context/index';
+import './HomePage.css';
+import { Link } from 'react-router-dom';
+
+export function HomePage() {
+  const { videos } = useVideos();
+  console.log(videos)
```
Comment (vaishnavipaliya11) [inline src/Pages/HomePage/HomePage.js]: can remove console

**Review by vaishnavipaliya11 — APPROVED** [review]
Please don't make so many file changes in a single PR.
Remove logs rest all seems good.

File: src/Pages/HomePage/HomePage.css (hunk 1 above)
Comment (omkarpatke) [reply src/Pages/HomePage/HomePage.css:29]: ok

Comment (omkarpatke) [reply src/Pages/HomePage/HomePage.css]: 👍

**Pull request merged**